src/accounts/models.py

```diff
@@ -67,7 +68,7 @@ class CompanyProfile(models.Model):
                                         through_fields=('company', 'student'), related_name='joboffers')
     internship_offers = models.ManyToManyField(StudentProfile, through='company.InternshipOffer',
                                                through_fields=('company', 'student'), related_name='internshipoffers')
-    contact = models.CharField(max_length=20)
+    contact = models.CharField(max_length=10)
```

Comment:
why did you do reduce this

Reply:
Provide a one-off default for the field.